includes/class-wc-stripe-helper.php

```diff
@@ -1851,4 +1865,35 @@ public static function get_refund_reason_description( $refund_reason_key ) {
 				return __( 'Unknown reason', 'woocommerce-gateway-stripe' );
 		}
 	}
+
+	/**
+	 * Checks if there are other Buy Now Pay Later plugins active.
+	 *
+	 * @return bool
+	 */
+	public static function has_other_bnpl_plugins_active() {
+		$other_bnpl_gateway_ids = [ self::OFFICIAL_PLUGIN_ID_AFFIRM, self::OFFICIAL_PLUGIN_ID_KLARNA ];
+		foreach ( $other_bnpl_gateway_ids as $bnpl_gateway_id ) {
+			if ( self::has_gateway_plugin_active( $bnpl_gateway_id ) ) {
+				return true;
+			}
+		}
+		return false;
+	}
+
+	/**
+	 * Checks if a given payment gateway plugin is active.
+	 *
+	 * @param string $plugin_id
+	 * @return bool
+	 */
+	public static function has_gateway_plugin_active( $plugin_id ) {
+		$available_payment_gateways = WC()->payment_gateways->payment_gateways ?? [];
+		foreach ( $available_payment_gateways as $available_payment_gateway ) {
+			if ( $plugin_id === $available_payment_gateway->id && 'yes' === $available_payment_gateway->enabled ) {
```

Comment:
Not requesting a change here. There's a [is_plugin_active](https://developer.wordpress.org/reference/functions/is_plugin_active/) that we can use but it requires the plugin path. Since this is a simple check, checking the status like you do here is good.

Reply:
Yes, I tried to use it before, but I couldn't make it work properly. If I remember correctly, I had issues trying to match the path. I can try again on another PR and consider deprecating this method later.